force_bdss/core_plugins/dummy/dummy_notification_listener/dummy_notification_listener_factory.py

```diff
@@ -0,0 +1,23 @@
+from traits.api import String
+
+from force_bdss.api import (
+    factory_id,
+    BaseNotificationListenerFactory)
+
+from .dummy_notification_listener import DummyNotificationListener
+from .dummy_notification_listener_model import DummyNotificationListenerModel
+
+
+class DummyNotificationListenerFactory(BaseNotificationListenerFactory):
+    id = String(factory_id("enthought", "dummy_notification_listener"))
+
+    name = String("Dummy Notification Listener")
+
+    def create_model(self, model_data=None):
```

Comment:
Why don't we set the default value of `model_data` to `{}` directly instead of checking if it's None and if it's the case then set it to `{}` ?